task/hiffy/src/common.rs

```diff
@@ -594,12 +596,14 @@ pub(crate) fn sprot_send_recv_retries(
             &mut *rval,
         ),
     )?;
-    match drv_sprot_api::MsgType::from(result.msgtype) {
+    let msgtype = drv_sprot_api::MsgType::from_u8(result.msgtype)
+        .ok_or(hif::Failure::FunctionError(1))?;
```

Comment:
Why make this change?  We previously were using `MsgType::from`, which converts unknown values into `MgsType::Unknown`, which takes the `_ => {` branch below, so the behavior should be the same.

Reply:
It looks like `MsgType::Unknown` non longer exists, so that's probably why.